go.mod

```diff
@@ -49,7 +49,10 @@ require (
 	github.com/bradleyjkemp/cupaloy/v2 v2.8.0
 )
 
-require github.com/kr/text v0.2.0 // indirect
+require (
+	github.com/golobby/container/v3 v3.3.1 // indirect
+	github.com/kr/text v0.2.0 // indirect
+)
```

Comment:
We can remove `tools.go`, and run `go mod tidy` to remove the `wire` related dependencies.